pkgs/tools/misc/qmk/default.nix

```diff
@@ -1,18 +1,18 @@
-{ lib
-, python3
-, fetchpatch
-}:
+{ lib, python3, fetchpatch, writeText }:
 
 let
   inherit (python3.pkgs) buildPythonApplication fetchPypi;
-in
-buildPythonApplication rec {
+  setuppy = writeText "setup.py" ''
```

Comment:
What is the purpose of this? Would be nice to add a comment.

Reply:
The `buildPythonApplication` requires a `setup.py` to work. It has been removed in the upstream repository in favor of `setup.cfg`. This wrapper read the config from `setup.cfg` to make it compatible with `buildPythonApplication`.